File: solutions/go/darts/1/darts.go
package darts

import "math"

func Score(x, y float64) int {
    r := math.Pow(math.Pow(x,2)+math.Pow(y,2),.5)
	score:=10
    switch {
        case r>10:
    		score=0
		case r>5:
    		score=1
        case r>1:
    		score=5
    }
	return score
}

File: solutions/javascript/darts/1/darts.js
//
// This is only a SKELETON file for the 'Darts' exercise. It's been provided as a
// convenience to get you started writing code faster.
//

export const score = (x, y) => {
  const d = (Math.abs(x)**2 + Math.abs(y)**2)**.5
  return d<=1 ? 10 : ( d<=5 ? 5 : (d<=10 ? 1 : 0) )
};
